spec/cinch-pax-timer_spec.rb

```diff
@@ -16,7 +16,7 @@
     it 'should return the next pax' do
       msg = make_message(@bot, '!pax')
       msg = get_replies(msg).first
-      expect(msg.text).to match(/PAX.+is (approximatly )?\d+ days from now/)
+      expect(msg.text).to match(/PAX.+is (approximatly )?\d+ days from now (No official date, yet)/)
```

Comment:
revert this, I leave the "no official date" bit off, so the tests don't fail when there is an official date